nextcord/abc.py

```diff
@@ -117,8 +117,17 @@ class User(Snowflake, Protocol):
     ----------
     name: :class:`str`
         The user's username.
+    global_name: :class:`str`
+        The user's global name.
+
+        .. versionadded:: 2.6
     discriminator: :class:`str`
         The user's discriminator.
+
+        .. warning::
+          This field is deprecated, and will only return if the user has not yet migrated to the
+          new [username](https://dis.gd/usernames) update.
+        ..deprecated:: 2.6
```

Comment:
```suggestion
        .. deprecated:: 2.6
```
<img width="794" alt="Screenshot 2023-06-16 at 11 43 49 PM" src="https://github.com/nextcord/nextcord/assets/50607143/483eac31-1016-4a75-9189-8e075c4bcad2">

The formatting here is leading to errors in the rendered docs.